docs/content/administration/reverse-proxies.en-us.md

```diff
@@ -112,7 +124,7 @@ server {
 
 Set `[server] STATIC_URL_PREFIX = http://cdn.example.com/gitea` in your configuration.
 
-```apacheconf
+```
```

Comment:
You can use `nginx` for all nginx config blocks.

Reply:
Done in 16cb5fc6aff0f24182535e4dff453810893f9589